examples/lavalink-basic-bot.rs

```diff
@@ -194,24 +199,69 @@ async fn play(msg: Message, state: State) -> anyhow::Result<()> {
 
     let loaded = serde_json::from_slice::<LoadedTracks>(&response_bytes)?;
 
-    if let Some(track) = loaded.tracks.first() {
-        player.send(Play::from((guild_id, &track.track)))?;
+    match loaded.data {
+        Track(track) => {
+            player.send(Play::from((guild_id, &track.encoded)))?;
```

Comment:
This could be simplified
```suggestion
    let track = match loaded.data {
        Track(track) => Some(track),
        Playlist(top_track) => playlist.tracks.first(),
        // ...
    };
    if let Some(track) = track {
        player.send(Play::from(guild_id, &track.encoded)))?;
        // ...
    }
```